feat: Allow setting of Git remote

nalysius opened the pull request:
Git is decentralised, so it's usual to have several remote
repositories. Origin is usually the default remote repository
but it's not mandatory.

Sentry-cli only allowed to use the remote repository "origin" when
working with commits during a release.

It is now possible to set the remote name in the configuration file
or as environment variable.
In ~/.sentryclirc: defaults.vcs_remote.
In env: SENTRY_VCS_REMOTE.
If not provided, defaults to "origin" so it's backward-compatible.

kamilogorek commented on src/config.rs:
```diff
@@ -546,3 +554,17 @@ fn get_default_log_level(ini: &Ini) -> Result<log::LevelFilter, Error> {
+/// origin.
+fn get_default_vcs_remote(ini: &Ini) -> String {
+    if let Ok(remote) = env::var("SENTRY_VCS_REMOTE") {
+        return remote.to_string();
```
Remove `return` statements to unify with the rest of the file

nalysius replied:
`return`s have been removed.

kamilogorek commented on src/utils/vcs.rs:
```diff
@@ -355,14 +367,19 @@ pub fn find_head() -> Result<String, Error> {
 
 /// Given commit specs and repos this returns a list of head commits
 /// from it.
-pub fn find_heads(specs: Option<Vec<CommitSpec>>, repos: &[Repo]) -> Result<Vec<Ref>, Error> {
```
The spec comment should be updated to mention remote as well.

nalysius replied:
Comment line 368 now speaks about remote.

kamilogorek commented:
Also can you update docs and mention that it should be merge once this change is? https://github.com/getsentry/sentry-docs/blob/master/src/collections/_documentation/cli/configuration.md

Thanks!

nalysius commented:
Thank you for your review @kamilogorek !

I've fixed the two points about the code, removed the `return`s and added the remote_name in the comment of the method. I've force-pushed to keep a clean commit history.
I will update the documentation too.
